### ISSUE TITLE
afx/wh csgo crash

### ISSUE THREAD

**Issue by dtugend** [advancedfx/advancedfx#606]
## Workaround

For old maps / demos only:

Please copy the afx/wh.cfg to your own.cfg file and **remove the following commands from the end**:

```
// Commands to improve entity detection (https://github.com/advancedfx/advancedfx/issues/607):
mirv_cvar_unhide_all
cl_modelfastpath 0
cl_tlucfastpath 0
cl_brushfastpath 0
```

Use the new cfg file for old maps, instead of afx/wh.cfg!

_However the wh will look a bit different (it will mostly affect dead player models color to be white instead of yellow / blue for some)._ 

## Background

This reverses the commands in afx/wh cfg added due to [mirv_streams: e.g. ragdolls not detectable by entity handle #607](https://github.com/advancedfx/advancedfx/issues/607).
Basically the commands expose an normally inaccessible code-path in CS:GO that is normally not used and has bugs that lead to the crash.

_Originally posted by @dtugend in https://github.com/advancedfx/advancedfx/discussions/565#discussioncomment-365759_
